packages/traceloop-sdk/tests/dataset/test_rows_operations_vcr.py

```
@@ -0,0 +1,91 @@
+        # Create a dataset to delete
+        unique_slug = f"test-delete-{int(time.time())}"
+        
+        csv_content = """Name\nTestData"""
+        
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".csv", delete=False) as f:
+            f.write(csv_content)
+            csv_path = f.name
+        
+        try:
+            # Create dataset
+            dataset = datasets.from_csv(
```

Ruff (0.12.2)

Local variable `dataset` is assigned to but never used

Remove assignment to unused variable `dataset`

(F841)